src/app/_components/page-content/DownloadCard.tsx

```diff
@@ -53,44 +94,44 @@ export const DownloadCard = ({ data, rows }) =>
             {data.image_de_la_carte.data?.attributes.url && (
               <div className='fr-tile__header'>
                 <div className='fr-tile__pictogram'>
-                  <img src={data.image_de_la_carte.data.attributes.url} />
+                  <img src={data.image_de_la_carte.data.attributes.url} alt='Illustration' />
```

Comment:
```suggestion
                  <img src={data.image_de_la_carte.data.attributes.url} alt='' />
```
`Illustration` n'apporte pas d'information essentielle sur le contenu à une personne utilisant des outils d'accessibilité